cpp/arcticdb/version/python_bindings.cpp

```diff
@@ -651,6 +653,13 @@ void register_bindings(py::module &version, py::exception<arcticdb::ArcticExcept
              "Returns latest timestamp of a symbol")
         ;
 
+    py::class_<ManualClockVersionStore, PythonVersionStore>(version, "ManualClockVersionStore")
```

Comment:
Did you know that there is a PilotedClock that just counts up monotonically from zero?

Reply:
I forgot about it, but looking at it, it's equivalent to the existing `LinearClock` where the "time" increase by 1 after each call.

Whereas the behaviour I need to test is the time stays the same between calls (to match the low-resolution clock on non-Linux).